source/op/descrpt_se_a_mask.cc

```diff
@@ -0,0 +1,342 @@
+
+#include <cmath>
+
+#include "ComputeDescriptor.h"
+#include "custom_op.h"
+#include "errors.h"
+#include "fmt_nlist.h"
+#include "neighbor_list.h"
+
+typedef double boxtensor_t;
+typedef double compute_t;
+
+REGISTER_OP("DescrptSeAMask")
+    .Attr("T: {float, double} = DT_DOUBLE")
+    .Input("coord: T")
+    .Input("type: int32")
+    .Input("mask: int32")
+    .Input("box: T")         // Not used in practice
+    .Input("natoms: int32")  // Not used in practice
+    .Input("mesh: int32")    // Not used in practice
+    .Attr("total_atom_num: int")
```

Comment:
It is difficult to understand the difference between `natoms` and `total_atom_num`. Is there a better name?

Reply:
Or at least add a doc to it.

Reply:
It's removed for now. Variable `total_atom_num` can also be fetched from `natoms`.